lib/verifyConditions.js

```diff
@@ -14,7 +14,11 @@ module.exports = async (pluginConfig, context) => {
     const registryHost = env.REGISTRY_HOST || pluginConfig.registry;
 
     if (registryHost && !pluginConfig.skipRegistryLogin && !pluginConfig.isChartMuseum && env.REGISTRY_USERNAME && env.REGISTRY_PASSWORD) {
-        const registryUrl = registryHost.replace("oci://", "").split('/')[0];
+        const registryUrl = registryHost
```

Comment:
The variable name is confusing, let's rename it

```suggestion
        const domain = registryHost
```